tests/JUnitOutputTest.cpp

```diff
@@ -296,7 +296,7 @@ TEST_GROUP(JUnitOutputTest)
 
     void setup()
     {
-        UT_PTR_SET(PlatformSpecificFOpen, (PlatformSpecificFile(*)(const char*, const char*))mockFOpen);
```

Comment:
Spurious cast; I think it shouldn't have been there to start with. The cl2000 doesn't like it